### ISSUE TITLE
Training Setting

### ISSUE THREAD

**Issue by lxtGH** [zhanghang1989/gluoncv-torch#5]
Hi ! Thanks for sharing the trained model, I wonder how to reproduce this result.
For example, for ADE dataset, this training set is the same as Pytorch-Encoding repo??
What is the learning rate and crop size, batch size

**Comment by zhanghang1989** [comment]
Please see GluonCV for training detail
